tests/test_prove.rs

```diff
@@ -51,8 +52,12 @@ fn assert_private_input_eq(
     expected_private_input: AirPrivateInputSerializable,
 ) {
     fn remove_file_keys(private_input: &mut AirPrivateInput) {
-        private_input.0.remove("trace_path");
-        private_input.0.remove("memory_path");
+        private_input
+            .0
+            .remove(&BuiltinName::from_str("trace_path").unwrap());
```

Comment:
Seems incorrect. `trace_path` and `memory_path` are keys inside the private input file that are not builtins, just paths to some files. 

Given the changes made recently to `cairo-vm`, you should be able to remove these `remove` statements. If it doesn't work for some reason then there's a bug in `cairo-vm`.

Also, don't use `unwrap()` in code that may fail, unless you're 100% sure it won't happen because of an invariant that you can't express in the type system (+ add a comment about why this invariant holds).